.ci/setup.sh

```diff
@@ -11,7 +11,7 @@ export JAVA_APT_PKG="oracle-java8-set-default"
 # install these testing packages we need
 if [[ "$TASK" == "rpkg" ]];
 then
-  Rscript -e "install.packages(c('data.table', 'devtools', 'futile.logger', 'knitr', 'lintr', 'testthat', 'rmarkdown', 'uuid'), repos = 'http://cran.rstudio.com')"
+  Rscript -e "install.packages(c('data.table', 'futile.logger', 'knitr', 'lintr', 'testthat', 'rmarkdown', 'uuid'), repos = 'http://cran.rstudio.com')"
```

Comment:
We'll need to explicitly declare purrr here too since it apparently was being installed because of devtools. Can you make sure all of our [dependencies](https://github.com/uptake/uptasticsearch/blob/master/r-pkg/DESCRIPTION) are called out here?

Reply:
ah! yep will do

Reply:
updated! hopefully it works. Scary how many of these things we were getting just because of `devtools` :grimacing:

Reply:
Apparently we need the Suggests too, like covr... may be worth deleting covr from Suggests actually since we don't use it in CI.

Reply:
I guess we need Suggests too like covr!

Reply:
I saw your comment in my email but I don't see it here...I agree we should just remove `covr` in `DESCRIPTION`. If we do that, we should probably do a minor release to CRAN after.

Remember the whole "`devtools::install()` installs all `Suggests` recursively" thing? :grimacing:

Reply:
Ha, sorry, I deleted it. I realized we *do* use covr in CI -- but we install it inside the script that calculates coverage. I was thinking it's better to leave it in Suggests and install it in CI.

Reply:
oohhhhh ok got it. Yeah I'll make sure to add it. Guess it's another thing we  were getting from `devtools`